schemas/orchestrator.yml

```
@@ -83,6 +83,19 @@
       description: >
         Type of resource being acted upon.
 
+    - name: resource.parent_type
+      level: extended
+      type: keyword
+      example: DaemonSet
+      description: >
+        Type or kind of the parent resource of the resource being acted upon.
+
+    - name: resource.ip
+      level: extended
+      type: ip
+      description: >
+        IP address of the resource being acted upon.
```

```suggestion
        IP address of the resource being acted upon.
     normalize:
       - array
```

I suggest we specify this field to expect an array of values.

I'm not that familiar with k8s networking, but I imagine there are dual-stack deployments where a pod could have IPv4 and IPv6 addresses? Or perhaps primary and secondary IPs are assigned to a single pod or deployment?

---

I believe you're correct. k8s does support dual-stack networking.